plugins/chat/assets/stylesheets/common/chat-thread-header-button.scss

```diff
@@ -0,0 +1,53 @@
+.open-thread-list-btn {
```

Comment:
As much as possible, we should aim for a 1-1 relation between component name and css class, it's way easier for designers to find their way